galasa-ui/src/tests/routes/auth.test.ts

```diff
@@ -1,8 +1,12 @@
-/**
+/*
  * Copyright contributors to the Galasa project
- * @jest-environment node
+ *
+ * SPDX-License-Identifier: EPL-2.0
  */
 
+/* 
+* @jest-environment node
```

Comment:
Formatting of this comment is off.

Reply:
@KirbyKatcher Is this actually needed ? https://jestjs.io/docs/configuration says that 
this kind of thing can be put into a configuration file rather than in each source file ?

And will having it in the second comment block in a file cause a problem ?